### ISSUE TITLE
Validation loss metric

### ISSUE THREAD

**Issue by OhadCohen97** [RetroCirce/HTS-Audio-Transformer#46]
Hi,

I am wondering why was the validation loss not tracked as a metric alongside validation accuracy? It's crucial and necessary for identifying overfitting during training. In your code, there are no logs for the validation loss. How can you tell while training if the model is overfitting? 

Thank you!
